tenets/codelingo/go/unused-private-functions/codelingo.yaml

```diff
@@ -0,0 +1,20 @@
+tenets:
+  - name: find-unused-private-funcs
+    doc: Finds unused private functions.
+    flows:
+      codelingo/review:
+        comment: This private function is not used in the package.
+    query: |
+      import codelingo/ast/go
+
+      go.func_decl(depth = any):
+        @review.comment
+        go.ident:
+          private == "true"
+          name as funcName
+          exclude:
+            name == "main"
```

Comment:
We should also exclude the `init` function https://golang.org/doc/effective_go.html?#init